content/en/blog/2023-02-01-january-update/index.md

```diff
@@ -0,0 +1,348 @@
+> will be a Kustomize Environment and the other a Helm-based environment.
+
+{{< youtube Bmh7kKYLIhY >}}
+
+> Flux Security & Scalability using VS Code GitOps Extension
+>
+> Recently Flux has released two new features (OCI and Cosign) for scalable and
+> secure GitOps. Juozas Gaigalas, a Developer Experience Engineer at Weaveworks,
+> will demonstrate how developers and platform engineers can quickly create
+> scalable and Cosign-verified GitOps configurations using VS Code GitOps Tools
+> extension. New and experienced Flux users can learn about Flux’s OCI and Cosign
+> support through this demo. Join us!
+
+Here is a list of additional videos and topics we really enjoyed -
+please let us know if we missed anything of interest and we will make
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+February - tune in to learn more about Flux and GitOps best practices,
+get to know the team and join our community.
+
+### Flux Bug Scrub
+
+Our Flux Bug Scrubs still are happening on a weekly basis and remain one
+of the best ways to get involved in Flux. They are a friendly and
+welcoming way to learn more about contributing and how Flux is organised
+as a project.
+
+The next dates are going to be:
+
+- [2023-02-02 18:00 UTC, 19:00 CET](/#calendar)
+- [2023-02-08 13:00 UTC, 14:00 CET](/#calendar)
+- [2023-02-16 18:00 UTC, 19:00 CET](/#calendar)
+- [2023-02-22 13:00 UTC, 14:00 CET](/#calendar)
+
+We are flexible with subjects and often go with the interests of the
+group or of the presenter. If you want to come and join us in either
+capacity, just show up or if you have questions, reach out to Kingdon on
+Slack.
+
+We really enjoyed this [demo of the k3d git
+server](https://www.youtube.com/watch?v=hNt3v0kk6ec)
+recently. It's a local Git server that runs outside of Kubernetes, to
+support offline dev in a realistic but also simple way that does not
+depend on GitHub or other hosted services.
+
+## In other news
+
+### Conference Call For Papers
+
+Conferences are all about the people. It's also more fun to present
+together. You get to share collective experience and be more entertaining
+as a duo!
+
+Two upcoming call for paper deadlines are the following
+
+- CFP until 2023-02-05, [SustainabilityCon](https://events.linuxfoundation.org/cdcon-gitopscon/program/cfp/)
+  > May 10 – 12, 2023 | Vancouver, Canada
+  > Join the community of developers, technologists, sustainability leaders
+  > and anyone working on technological solutions to decarbonize the global
+  > economy, mitigate and address the impacts of climate change, and build a
+  > more sustainable future. SustainabilityCon provides a forum to drive open
+  > source innovation in energy efficiency and interoperability and clean
+  > development practices within industries ranging from manufacturing to
+  > agriculture and beyond through collaboration and learning within the community.
```

Comment:
@nikimanoledaki I thought I'd mention the CFP here as well - WDYT?

Reply:
LGTM! Thank you for the shoutout!